subsys/net/ip/net_mgmt.c

```diff
@@ -39,8 +39,8 @@ static K_SEM_DEFINE(net_mgmt_lock, 1, 1);
 K_KERNEL_STACK_DEFINE(mgmt_stack, CONFIG_NET_MGMT_EVENT_STACK_SIZE);
 static struct k_thread mgmt_thread_data;
 static struct mgmt_event_entry events[CONFIG_NET_MGMT_EVENT_QUEUE_SIZE];
-static uint32_t global_event_mask;
-static sys_slist_t event_callbacks;
+static uint32_t global_event_mask = 0U;
```

Comment:
Strictly speaking, the static value is initialized to 0 automatically so this global_event_mask init is not really needed,  but as it does no harm either.

Reply:
Will fix, as it is a compliance error.